python_modules/dagster/dagster/_core/definitions/repository_definition/repository_data_builder.py

```diff
@@ -417,8 +438,8 @@ def build_caching_repository_data_from_dict(
     )
 
 
-def _process_and_validate_target(
-    schedule_or_sensor_def: Union[
+def _process_and_validate_target_job(
+    automation_def: Union[
```

Comment:
in our internal code paths at the moment do we not say "instigator" more? I'd rather stay consistent with that for now

Reply:
older codepaths definitely use "instigator". I was thinking `automation_def` here for consistency with `AutomationTarget` but on reflection I think you're right and we should just use `instigator_def` everywhere.

Reply:
oops forgot to make it but just pushed it